staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -148,7 +150,11 @@ type Config struct {
 	PostStartHooks map[string]PostStartHookConfigEntry
 
 	// Version will enable the /version endpoint if non-nil
-	Version *version.Info
+	// Deprecated: Use EffectiveVersion instead
+	Version          *apimachineryversion.Info
```

Comment:
this appears to not work anymore (I cannot find references to it).  If it no longer works, it should be deleted. Deprecation means, "still works, use replacement", not "no longer works".

Reply:
Since the `Version` seems to be more about the build version with all the GitCommit info, decided to restore it back.